test/ci-scripts/Write-TestResults.ps1

```diff
@@ -441,6 +468,15 @@ switch ($CredScanResult) {
     }
 }
 
+switch ($TemplateAnalyzerResult) {
+    "PASS" { $TemplateAnalyzerResultColor = "brightgreen" }
```

Comment:
Not sure what color this is and that in other areas this is used, but is this an accessible color? Does it have enough contrast?

Reply:
you can see a sample here: https://github.com/Azure/azure-quickstart-templates/tree/master/100-blank-template - the color used for badges in github.